### PR TITLE
Corrected version of the Online Tutorial

### PR DESCRIPTION
**Pull request by schaarj**
* In general line breaks in the command boxes are sometimes not retained. I haven’t really figured out what causes specific line braeks to be ignored and just enforced the ignored ones I observed with \<br\>, not sure whether there is a better solution but an option might be to switch from \<a\> blocks to {% highlight %} which would also add syntax highlighting (python for pymol and bash for command line). 
* I also added the notes I made regarding some additional comments/suggestions while following the tutorial below
* While most is stated in the introductory text an additional short bullet list with requirements for the tutorial might be neat including the web services that require subscription 

# Simulation part

- Is the initial secondary structure of the peptide supposed to be distributed among students (following the instructions only helix will be generated)
	- adjust tutorial to be ambiguous?
- Is there actually somewhere a precomputed set of the 50 ns runs of different starting conformations available. Might be nice especially for people without access to a cluster! (I now have the trajectories of on completed run for all three conformations)
- At some points results are supposed to be inspected in pymol yet conversion of .gro to .pdb or alternative approach is not mentioned
	- is this supposed to be a challenge? (I went the conversion to pdb route even though pymol recognised the gromacs file in some way, but since getting pymol to open the files wasn’t straightforward I refrained from further trying to open the files without conversion to pdb)
- xvg_plot.py fails due to issues with the bg-color (works when adding “-b white” option) -> fix the script or command lines?
- .mdp files are read only in /opt/data/mdp/ but supposed to be edited —> change permissions or add instructions to copy files to working directory
- is it reasonable to combine energies, temperature, and pressure in a single plot (scaling?) which is what you get when you follow the instructions
- Maybe add a hint to how the graphs are supposed to look like somewhere in the text, or should this be answered by the students (guided by the current question set)
- maybe add instructions to visualise trajectories of mimnimizations or equilibrations (see constraints of heavy atoms)?
- html formatting destroys multiple whitespaces in sed commands I haven’t found a workaround that keeps the formatting and works with copy/paste of the command line
	- added escapes before each whitespace according to Alexandres suggestion -> now copy/pasteable
- gmx dump command fails with .tpr file in contrast to instructions 
- Asked to extract 100 frames but command gives timestep of 50 (~1000 frames) which one, extract 100 or 1000 frames?
- instead of dumping 1000 pngs maybe add freemol plugin to vm and export directly to movie?
- the section “Convergence of the thermodynamical parameters” obviously refers to an old manual of gromacs (4.???) either a link (should be specific for the version referenced and not to a current version which might be updated with each gromacs release) or the manual referred to should be provided
	- without further details this section appears to be hard to follow to me! Instruction to view graphs in xmgrace is challenging for someone not familiar with the program (a link to a decent manual/tutorial specifically for MD data would com in handy) Without further instructions on how to use xmgrace with the data, there doesn’t seem to be any added value. Mentioning the advantages of xmgrace and sticking to viewing graphs with the xvg_plot.py seems more reasonable to me but maybe I’m missing something!

- “unphysical self-interactions” - maybe find a better term

## Conformational dynamics and stability II – Root Mean Square Fluctuation (RMSF)
- colouring by b factor seems more meaningful for the md_temperature-factors-residue.pdb structure - mixup in pymol command? 

## Conformational dynamics and stability III – Root Mean Square Deviation (RMSD)
- Extent guidance in instructions or by comments above the commands that the commands are nearly identical and that the different output filenames expect the user to make different choices in the prompts (Took me some time to figure out the intention of this part)
- Maybe extend this part a little e.g. what values might be acceptable… 

- dssp (from within pymol) does not seem to work within the VM 
- maybe add some more information on clustering output files or a reference where to find it (gromacs manual)

# Docking
The part about submitting to the Haddock web server was somewhat confusing to me as it appeared as some instructions on what to select and fill in were in the text while others where in the boxes Since filling in the Form is one step it might be better to describe the relevant options in the text and than have one combined box with all the input and choices to be made in one place.
Unfortunately the test run on the webserver failed in the clustering step so I didn't go through the evaluation part. The log files weren't conclusive about what exactly went wrong and double checking input and resubmitting didn't help either. I will try to rerun it locally and hopefully track down why the run didn't succeed. The data of the failed run should be accessible under: http://milou.science.uu.nl/serviceresults/HADDOCK2.2/9246725127/p53_mdm/

### CODE REVIEW

**Comment by JoaoRodrigues** [comment]
Wow, that's a lot of work. Thanks Jörg, I'll go over the changes carefully,
please, don't merge yet because of the two 'merge' commits in there.

A seg, 25/01/2016, 06:38, Joerg Schaarschmidt <notifications@github.com>
escreveu:

**Comment by amjjbonvin** [comment]
I leave it to you to handle Joao!

=================================
 Prof. A.M.J.J. Bonvin | Faculty of Science
 Utrecht University | bonvinlab.org 
 a.m.j.j.bonvin@uu.nl | T: +31 30 2533859 
=================================

**Comment by schaarj** [comment]
@JoaoRodrigues  if you want we can also Skype so I can walk you throughgh some of the comments my Skype name is J.Schaarschmidt
Best Joerg

**Comment by amjjbonvin** [comment]
Merged manually - resolving conflicts

**Comment by JoaoRodrigues** [comment]
@amjjbonvin @schaarj @adrimel

I simplified the commits and applied most of the changes to the tutorial. Commits should really be either on a single file, or a major change across all files that represents one simple goal. Multiple changes here and there to multiple files in one commit are not good practice. It's extremely hard to revert if necessary, for example.

I will go over the suggestions in the body of the pull request message later. I also left the docking tutorial for later, because it's the one needing more attention. Also, @schaarj , can you send me an email (or slack) with the output of some of the errors described above (e.g. the xvg_plot.py script)?
